.vscode/extensions.json

```diff
@@ -0,0 +1,14 @@
+{
+	// See https://go.microsoft.com/fwlink/?LinkId=827846
+	// for the documentation about the extensions.json format
+	"recommendations": [
+		"yzhang.markdown-all-in-one",
+		"svsool.markdown-memo",
+		"bierner.markdown-checkbox",

```

Comment:
What is this?

Reply:
It's https://marketplace.visualstudio.com/items?itemName=bierner.markdown-checkbox

Removed now, along with a few others; they came from https://github.com/srid/emanote-template/tree/master/.vscode